src/app/forms/templates/psv/psv-notes.template.ts

```diff
@@ -12,7 +13,17 @@ export const PsvNotes: FormNode = {
       type: FormNodeTypes.CONTROL,
       viewType: FormNodeViewTypes.STRING,
       editType: FormNodeEditTypes.TEXTAREA,
-      validators: [{ name: ValidatorNames.MaxLength, args: 1024 }]
+      validators: [{ name: ValidatorNames.MaxLength, args: 1024 }],
+
+      // Obviously we don't want to add functions in the JSON templates, but it could be handled using a map, the same way other validators are handled
+      warningValidators: [
+        (control: AbstractControl): ValidationErrors | null => {
+          if (control.value.includes('foobar')) {
+            return { test: 'should not equal foobar' };
+          }
+          return null;
+        }
+      ]
```

Comment:
then we just add warning validators the exact same way we add normal validators, all through templates.